src/test/scala/mesosphere/marathon/api/v2/ModelValidationTest.scala

```diff
@@ -103,7 +103,7 @@ class ModelValidationTest
       case f: Failure =>
         val errors = (Json.toJson(f) \ "details").as[Seq[JsObject]]
         errors should have size 1
-        (errors.head \ "path").as[String] should be("/groups(0)/apps(1)")
+        (errors.head \ "path").as[String] should be("/groups(group1)/apps(invalid)")
```

Comment:
I think the full path would be a lot more informative than the relative one. 

In the example above, I'd expect to receive an error pointing me to `/group1/invalid` so that I can understand immediately what the error refers to. 

@aquamatthias will this ship in a 1.2 version? If so we'll have to schedule some time to adjust the UI.